src/nvim/tui/terminfo.c

```diff
@@ -34,6 +34,18 @@ bool terminfo_is_term_family(const char *term, const char *family)
     && ('\0' == term[flen] || '-' == term[flen]);
 }
 
+bool terminfo_is_screen_host_term_family(const char *term, const char *family)
```

Comment:
isn't it possible to reuse terminfo_is_term_family passing `term[7]` instead of `term` as first parameter ?

Reply:
I changed it to that. For this to work, however, https://github.com/neovim/neovim/pull/12005/commits/c2d288e293a8fa8acba72d73a825213b8a1d280c (#12005) is required.